iib/workers/tasks/build.py

```diff
@@ -652,6 +652,41 @@ def _overwrite_from_index(
             temp_dir.cleanup()
 
 
+def get_index_image_info(overwrite_from_index_token, from_index=None, default_ocp_version='v4.5'):
+    """
+    Get arches, resolved pull specification and ocp_version for the index image.
+
+    :param str overwrite_from_index_token: the token used for overwriting the input
+        ``from_index`` image. This is required for non-privileged users to use
+        ``overwrite_from_index``. The format of the token must be in the format "user:password".
+    :param str from_index: the pull specification of the index image to be resolved.
+    :param str default_ocp_version: default ocp_version to use if index image pull_spec is absent.
+    :return: dictionary of resolved index image pull spec, set of arches and default ocp_version
+    :rtype: dict
+    """
+    result = {
+        'resolved_from_index': None,
+        'ocp_version': default_ocp_version,
+        'arches': set(),
+        'resolved_distribution_scope': 'prod',
```

Comment:
Should this be configurable?

Reply:
yeah, there's a separate issue for it and the default should be `prod` anyways.